Source/WebCore/Modules/screen-wake-lock/WakeLockManager.h

```diff
@@ -39,9 +39,12 @@ class Document;
 class SleepDisabler;
 class WakeLockSentinel;
 
-class WakeLockManager final : public VisibilityChangeClient {
+class WakeLockManager final : public RefCounted<WakeLockManager>, VisibilityChangeClient {
     WTF_MAKE_TZONE_ALLOCATED(WakeLockManager);
 public:
+    void ref() const final { RefCounted::ref(); }
```

Comment:
The WakeLockManager is owned by the document, no need to subclass RefCounted, it can forward its ref-counting to its document. 
```
void ref() const final { m_document.ref(); }
void deref() const final { m_document.deref(); }
```

Reply:
Fixed.